components/account/AccountButton.tsx

```diff
@@ -77,34 +76,50 @@ const AccountButton: FC<{
             )}
           </div>
         ) : (
-          <div className="flex h-ztg-40">
+          <div
+            className={`flex flex-1	items-center justify-end h-full rounded-full cursor-pointer ${
+              pathname === "/" ? "bg-transparent border-white" : "border-black"
+            }`}
+            onClick={() => {
```

Comment:
Connect wallet button needs contrast
<img width="1427" alt="image" src="https://user-images.githubusercontent.com/4950844/221163427-d00ff755-a193-4917-951e-51c5c8b9dffa.png">

Reply:
I think it disappears when it's loading...I'll see too adding some sort of skeleton for it instead

Reply:
No it is there, just back text on black background

Reply:
Found it! I made some changes to the styling and the layout so it isn't so janky when it's loaded